### ISSUE TITLE
Add a normative rule for HTML LS §4.8.4.4: img must have alt attribute

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#3964]
## Summary

markuplint has no *normative* HTML LS §4.8.4.4 rule requiring `<img>` to carry an `alt` attribute. The only rule that can catch a fully alt-less `<img>` is `require-accessible-name`, an ARIA-heuristic rule that is `non-normative` and only ships via the `markuplint:a11y` preset — which `tests/external/bench/config.ts` intentionally does not extend (the bench is scoped to HTML LS / WAI-ARIA / URL LS conformance rules, not accessibility best-practice heuristics).

## Repro

```html
<figure><img src="image.jpg"></figure>
```

nu-validator: error — `An "img" element must have an "alt" attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.`

markuplint (all bench rules enabled): 0 violations.

Fixture: `tests/external/validator/tests/html/elements/img/missing-alt-in-figure-novalid.html` (`nu-only`).

## Spec basis

[HTML LS §4.8.4.4 "The alt attribute"](https://html.spec.whatwg.org/multipage/images.html#alt) lists ~13 exception scenarios (link content, decorative image, image is key part of content with a caption, etc.) under which `alt` may be omitted; in every other case it must be present. Being inside a `<figure>` (with or without `<figcaption>`) is **not** one of the listed exceptions — the only figure/figcaption-related normative text is in the *caption information* extraction algorithm (used to derive fallback text when the image cannot be rendered), which presupposes `alt` is already required, not exempted:

> If the image is a descendant of a `figure` element that has a child `figcaption` element, and, ignoring the `figcaption` element and its descendants, the `figure` element has no flow content descendants other than inter-element whitespace and the `img` element, then return the contents of the first such `figcaption` element.

## Current implementation

- No rule directly mirrors §4.8.4.4's `alt`-presence requirement.
- `require-accessible-name` (`packages/@markuplint/rules/src/require-accessible-name/index.ts`) does functionally catch this case — role `img` has `accessibleNameRequired: true`, and `getAccessibleName()`'s `handleImg` (`packages/@markuplint/ml-spec/src/algorithm/aria/accname/element-names.ts`) returns `null` when both `alt` and `title` are absent — but it ships only through `packages/@markuplint/config-presets/src/preset.a11y.jsonc`, a preset the bench does not extend by design.
- `packages/@markuplint/html-spec/src/spec.img.jsonc` defines `alt`'s conditional-required/implicit-role interplay (`alt=""` → `presentation` role) but nothing enforces plain presence.

## Proposed direction

Two non-exclusive options:

1. Add a small dedicated normative rule (e.g. `require-img-alt`) that mirrors HTML LS §4.8.4.4's default case: `<img>` without an `alt` attribute is a violation, full stop — deliberately *not* attempting to model all ~13 exception scenarios (those are guidance for *when omission is acceptable*, not machine-checkable preconditions in most cases). This keeps the rule's scope narrow and keeps it independent of the `a11y` preset's heuristic/ARIA machinery, so it can be added to the bench.
2. Alternatively, special-case `require-accessible-name` to be included in the bench for this specific class of violation — rejected as a general approach since the bench's whole design principle is "HTML LS conformance rules only," and `require-accessible-name` covers many roles/elements beyond what HTML LS mandates.

Option 1 is the more bench-aligned direction. Existing pattern to follow: rules like `map-id-name-match`, `form-attr-references-form` are single-assertion normative rules with `specConformance: "normative"` preset entries in `packages/@markuplint/config-presets/src/preset.html-standard.jsonc`.

## Files

- `packages/@markuplint/rules/src/` (new rule directory)
- `packages/@markuplint/rules/src/index.ts`
- `packages/@markuplint/rules/schema.json`
- `packages/@markuplint/config-presets/src/preset.html-standard.jsonc`
- `tests/external/bench/config.ts` (enable once implemented)

## Bench impact

1 fixture currently `nu-only`. Resolving this Issue is expected to flip it to `match-error`.

**Comment by YusukeHirao** [comment]
Duplicate of #3944 — closing in favor of the existing, more thorough Issue. Added a factual correction there about require-accessible-name's actual behavior on this fixture.
